packs/feats/explosion-of-power.json

```diff
@@ -118,6 +118,7 @@
                 "key": "DamageAlteration",
                 "mode": "override",
                 "predicate": [
+                    "explosion-of-power",
```

Comment:
Would using selector of `"{item|id}-inline-damage"` work instead?

Reply:
This damage is being rolled from a spell Item Alteration, so it would have the spell's id and slug instead. In the localisation file, the inline damage itself has the `explosion-of-power` roll option.